src/main/java/crawlercommons/sitemaps/SiteMapParser.java

```diff
@@ -106,18 +111,43 @@ public boolean isStrict() {
 
     /**
      * @return whether the parser allows any namespace or just the one from the
-     *         specification
+     *         specification (or any namespace accepted,
+     *         {@link #addAcceptedNamespace(String)})
      */
     public boolean isStrictNamespace() {
         return strictNamespace;
     }
 
     /**
      * Sets the parser to allow any namespace or just the one from the
-     * specification
+     * specification (or any namespace accepted,
+     * {@link #addAcceptedNamespace(String)})
      */
     public void setStrictNamespace(boolean s) {
         strictNamespace = s;
+        if (strictNamespace) {
+            addAcceptedNamespace(Namespace.SITEMAP);
+        }
```

Comment:
I can see why you auto-add the default sitemap namespace, but side-effects to what looks like a straight-up POJO setter should be clearly called out. I'd suggest documenting that in the function Javadoc (`setting this to true will always add Namespace.SITEMAP`)

Reply:
It was merely to keep the API unchanged. I'll update the Javadoc.